tests/test.rs

```diff
@@ -507,33 +507,3 @@ fn gnu_apple_darwin() {
         cmd.must_not_have("-isysroot");
     }
 }
-
```

Comment:
These tests didn't require GCC, just check that the target-specific stuff works. They shouldn't be deleted.

There should also be a test that clang works and has everything set appropriately.